# Correspondences between protein sequence and structure
_____
In the 1960s, Christian Anfinsen postulated that the unique three-dimensional structure of a protein is determined by its amino acid sequence (sequence–structure–function paradigm). However, intrinsically disordered regions does not conform to this postulate. Disordered regions contribute to protein function and do not fold into a defined tertiary structure.

Protein structure prediction techniques fall into two main categories: Template-free (or de novo) and Template-based (or homology-modeling). The first one do not use any known structures. Useful when not a single structure in a protein family is known. Whereas the Template-based employ the similarity to another protein whose three-dimensional structure is known. 


_____
## Exercise 1
We will perform a template-based structure prediction
1. Prepare the fasta sequence of your protein of interest. If you´re interested in predict the disorder in a PDB entry, go to the main page [3cax](https://www.ebi.ac.uk/pdbe/entry/pdb/1zr9).
2. In the main page, you will find the grey box "Structure analysis" in the lower right side. Click on [Molecule details](https://www.ebi.ac.uk/pdbe/entry/pdb/1zr9/protein/1). In the upper gray box, identify the term "UniProt:" and copy the code (O00488).
3. Go to the [UniProt site](https://www.uniprot.org/) and search fot the UniProt code (O00488)[https://www.uniprot.org/uniprot/O00488]. Search for the (fasta sequence)[https://www.uniprot.org/uniprot/O00488.fasta] and copy it in the text editor installed in your computer.
4. Direct to the (IUPred2A)[https://iupred2a.elte.hu/], an intrinsic disordered predictor. 

 ![3cax_visualization_domain_pdbe](https://github.com/Claualvarez/ECCB2020/blob/master/Figures/3CAX_domainpdbeCATH.png?raw=true)


_______
## Exercise 2
Predict intrinsically disordered regions in our protein of interest
 1. Go to th
_______
## Further reading
 












